src/main.rs

```diff
@@ -149,5 +208,6 @@ impl<W: std::io::Write> GuessingGame<W> {
 }
 
 fn main() -> Result<(), std::io::Error> {
-    GuessingGame::new(100, rand::thread_rng(), std::io::stdout()).start()
+    // GuessingGame::new(100, rand::thread_rng(), std::io::stdout()).start()
+    RockPaperScissorsGame::new(rand::thread_rng(), std::io::stdout()).start()
```

Comment:
v prvej verzii by som kludne pridal prvotnu otazku, ze ci chces hrat guessing alebo RPS a nech sa user rozhodne. Jeden input navyse, si "backwards compatible". Znova pripominam, ze sa snazis mat po kazdom commite nieco zmysluplne.

Btw, v ramci tohto iterative developmentu celkom casto robis `git rebase -i`, prave preto, aby si tie commity v minulosti spravil viac zmysluplne. Predstav si to ako slidy. Ked si nejaky zle spravil, tak ho tak nenechas. Ked dava vacsi zmysel mergnut dva dokopy, tak to urobis. Je to tvoja prezentacia, reviewer (a hocijaky iny developer v buducnosti) je tvoje publikum.